contracts/NFTVesting/LinearVestingNFTTimeLock.sol

```diff
@@ -83,25 +83,43 @@ contract LinearVestingNFTTimeLock {
     function releaseTime() public view virtual returns (uint256) {
         return _releaseTime;
     }
+    /**
+     * @dev Returns the max duration that a token is allowed to vest
+     */
+    function maxDuration() public view virtual returns (uint256) {
+        return _maxDuration;
+    }
+    /**
+     * @dev Returns the max discount allowed for a token in percentage.
+     */
+    function maxDiscount() public view virtual returns (uint256) {
+        return _maxDiscount;
+    }
+    /**
+     * @dev Returns duration that NFT has been locked and vesting
+     */
+    function vestedDuration() public view returns (uint256) {
+        return block.timestamp - _releaseTime;
+    }
 
-     /**
+    /**
      * @dev Returns discount percentage for achieved from vesting
      */
 
-    function vestedDiscountPercentage() public view returns (uint256) {
+    function getDiscountPercentage() public view returns (uint256) {
         if (block.timestamp < _releaseTime) {
             return 0;
         }
-        return _maxDiscount * (block.timestamp - _releaseTime) / _maxDuration;
+        return (_maxDiscount * vestedDuration()) / maxDuration();
```

Comment:
What do you think about using brackets to group both the duration variables together instead?

Reply:
refactored this function to remove this